src/dropdown/src/Dropdown.tsx

```diff
@@ -55,7 +56,7 @@ DropdownIgnoredOption
     return node.key
   },
   getDisabled (node) {
-    return node.disabled === true
+    return (node as DropdownOptionBase).disabled === true
   },
```

Comment:
It's `DropdownOption`, not `DropdownOptionBase`.

When doing force cast, you need to know, the type annotation should follow the runtime behavior.

A not accurate cast is worse than an `any` cast.